src/pages/inventory/tabs/wrapperConfig/CustomDropDown.js

```diff
@@ -169,7 +182,7 @@ const CustomDropDownComponent = ({ options, label, isHidden, backgroundColor, ty
       hiddenSelectedRows = [],
       totalRowCount = 0,
     } = context;
-    if (enabledWithoutSelect && totalRowCount <= 4000) {
+    if (true) {
```

Comment:
@Senait7-ux After review comments, Do we need id condition if this is always true?